src/test/resources/com/aws/greengrass/clientdevices/auth/config_with_ec_ca.yaml

```diff
@@ -3,7 +3,7 @@ services:
   aws.greengrass.clientdevices.Auth:
     configuration:
       certificateAuthority:
-        ca_type: ["ECDSA_P256"]
+        caType: ["ECDSA_P256"]
```

Comment:
Changed to from `ca_type` to `caType` to adhere to our config variable naming convention (old `ca_key` outside of `certificateAuthority` will still be managed